app/src/main/java/org/liberty/android/fantastischmemo/ui/StatisticsScreen.java

```diff
@@ -246,6 +257,8 @@ public Chart generateChart(BarData data) {
             chart.getXAxis().setTextColor(Color.WHITE);
             chart.getAxisLeft().setTextColor(Color.WHITE);
             chart.getAxisRight().setTextColor(Color.WHITE);
+            chart.getAxisLeft().setValueFormatter(new ChartYAxisValueFormatter());
```

Comment:
Sorry not being clear. I mean to put ChartYAxisValueFormatter as a static member in StatisticsScreen and all AsyncTask should be able to reference. Then the initialization for each time to use it will be eliminated.